src/components/contactForm.js

```diff
@@ -130,20 +128,26 @@ const ContactForm = () => {
             <textarea
               id="message"
               name="message"
+              aria-controls="previewDiv"  // Added aria-controls
               onChange={handleChange}
               value={formState.message}
               placeholder="Message"
               required={true}
             />
           </div>
-
           <button className="btn btn-md primary min-130" type="submit">
             {contactFormBtn}
           </button>
         </form>
+        <div id="infoDiv">
+            {/* ... Info related to the inquiryType ... */}
```

Comment:
what this empty div are for?

Reply:
This comes from ADA issues. I think is a false positive. Usually that div should have content, but I don't think is necessary. I just want to try if doing this removes the issue about accessibility.